### PR TITLE
raidboss: convert 05-shb/eureka to TS

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts [hunk 1]
```diff
@@ -100,7 +123,7 @@ export default {
       preRun: (data) => data.gloryOfBozjaCount = (data.gloryOfBozjaCount || 0) + 1,
```
Comment (panicstevenson) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts:123]:
```suggestion
      preRun: (data) => data.gloryOfBozjaCount = (data.gloryOfBozjaCount ?? 0) + 1,
```

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts [hunk 2]
```diff
@@ -646,15 +697,15 @@ export default {
...
+          ids: data.seekerCometIds?.slice(0, 2),
         });
 
         if (cometData === null) {
           console.error('Baleful Comet: null cometData');
           return;
         }
         if (!cometData.combatants) {
-          cometData.error('Baleful Comet: null combatants');
+          console.error('Baleful Comet: null combatants');
```
Comment (panicstevenson) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts:708]: (@_@)

**Review by panicstevenson — APPROVED** [review]

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts [hunk 3]
```diff
@@ -1040,50 +1118,53 @@ export default {
...
       // Yellow tether between Knight and Warrior gives them a Physical Vulnerability Down debuff.
-      infoText: (_data, _matches, output) => output.text(),
+      infoText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
           en: 'Remove yellow; apply purple',
           de: 'Entferne Gelb; nimm Lila',
           fr: 'Retirez le jaune; appliquez le violet',
-          jp: 'スチールを切り、スペルを使用',
+          ja: 'スチールを切り、スペルを使用',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.ts:1134]: lol, didn't notice this.
Thanks you TypeScript!

File: ui/raidboss/data/05-shb/eureka/bozjan_southern_front.ts [hunk 4]
```diff
@@ -175,11 +191,11 @@ export default {
           // Stop any active timelines.
           data.StopCombat();
           // Prevent further triggers for any active CEs from firing.
-          data.ce = null;
+          delete data.ce;
```
[the remark this answers is not in the document]
Comment (quisquous) [reply ui/raidboss/data/05-shb/eureka/bozjan_southern_front.ts:194]: I don't have a preference here.  I think delete and `= undefined` are equivalent to me.

**Pull request merged**